src/shared/components/gene-search-panel/GeneSearchPanel.module.css

```diff
@@ -66,27 +67,26 @@
   border-bottom: 1px solid var(--color-medium-light-grey);
 }
 
-/* Species column */
-.resultsTable td:first-child {
-  width: 60%;
+.resultsTable td:not(:last-child) {
   vertical-align: top;
 }
 
-.speciesCount {
-  font-weight: var(--font-weight-bold);
-  padding: 0 0.5ch;
+.speciesName [data-part="common-name"] {
+  font-size: 14px;
```

Comment:
Two questions:

- Did you get the 14px from Andrea? I am surprised, because in other tables she uses 13px for common or scientific name
- Do we expect a cell with full species name to appear in other tables, or is this a one-off? Or do we postpone this question until we actually start seeing such species names appearing in other tables?